master/buildbot/test/unit/test_steps_shell.py

```diff
@@ -71,6 +72,11 @@ def test_constructor_args_validity(self):
                     lambda: shell.ShellCommand('build', "echo Hello World",
                             wrongArg1=1, wrongArg2='two'))
 
+    def test_describe_with_command_Interpolate(self):
```

Comment:
Right now, you are also testing how casting work in Interpolate class. I think you should separate command to a string variable, and pass it as an argument to `Interpolate` class and also assert that `self.describe()` return this string in array form.

Reply:
Done. We check `expected_output` with an output from `step.describe()`